### PR TITLE
Chaplain is no longer able to use guns (magical staves are permitted)

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

For c*ncil

# Changelog

:cl:  
tweak: Chaplains are no longer able to use guns, but can still use wands
/:cl:

### CODE REVIEW

**Comment by Maia-J** [comment]
What is the reasoning behind this even

**Comment by Moltijoe** [comment]
I like the idea, but the chaplain will need more stuff to support this playstyle in the event of something like nukies

**Comment by UltraBigRat** [comment]
What exactly is wrong with chap using a gun? If they have one they are either antag (should absolutely not apply gun restriction to chap antag) or theres enough of a threat for them to get one (wizard/big cult) meanwhile those antags can easily just pick off the chaplain by yoinking a gun with their insane mobility. This just reduces chaplain to a mediocre antimagic bonking stick for combat. 

Even outside of that, chaplain is now directly worse then any other job for combat for their only saving grace becomes  a mediocre antimagic bonking stick that they are now objectively the worst at using against antags compared to anyone with a gun.

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 1]
```diff
@@ -139,13 +139,18 @@
 /mob/living/carbon/human/can_use_guns(obj/item/G)
 	. = ..()
 
+	if(job == "Chaplain" && !istype(G, /obj/item/gun/magic)) // Can still use magic staves
```
Comment (Bibby0110) [inline code/modules/mob/living/carbon/human/human_helpers.dm]: add a trait

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

**Comment by neeshacark** [comment]
I will not publicly give the reasoning for this since Ynot didn't but it is a tradeoff that I think is worth it for what chap gets out of it.

**Comment by Moltijoe** [comment]
> I will not publicly give the reasoning for this since Ynot didn't but it is a tradeoff that I think is worth it for what chap gets out of it.

expecting people to like a sizable nerf without knowing about a potential follow-up buff is a bit hopeful

**Comment by ynot01** [comment]
this PR is for council's easy access and code review, not public opinion

the reactions are pretty funny though

**Review by Moltijoe — APPROVED** [review]
Code checks out, personally don't care for the flavour text sent in the to_chat but I can't put together what i'd think is better

**Comment by ynot01** [comment]
flavor text was run by skrem before posting

**Comment by Moltijoe** [comment]
> flavor text was run by skrem before posting

"Justice" feels more inquisitorial than standard holy

**Comment by ynot01** [comment]
its a mgr reference

**Comment by Moltijoe** [comment]
> its a mgr reference

fair enough

**Comment by RG4ORDR** [comment]
Lmao. Skill issue.
 didn't ask + ratio + you fell off + cope + seethe + mald + dilate + L + hoes mad + W + cry about it + stay mad + touch grass + pound sand + skill issue + quote tweet + get real + no bitches? + banned + cancelled + suspended + terminated + deleted + kys + you're* + not funny didn't laugh + radio + the audacity + go outside + cringe + get clapped + triggered + ggez + cap + whipped and nae nae'd + i own u + fatherless + motherless + brotherless + motherless + familyless + what the dog doin + reported + blame will deez + it wasn't me + unfunny + rejected + invalid opinion + clout seeker + get rekt + who asked + dead channel + the hood watches ishowspeed now + caught in 4k + ligma + retweet + benched + wrenched + boomer + did your mom + clown + basic + get good + ad hominem + typo + repelled + ur pp smol + drink bleach + redpilled + unsubbed + wait i never was subbed in the first place + lol + simp + sussy baka + straight cash + free + freer than a private dark auction + dogwater + ricewater + catwater + potatowater + onionwater + zero earned + zero pr + zero awareness + zero braincells + you're lost + more lost than a blind kid in a mcdonalds playplace + added to my cringe comp + nft owner + sex offender + sex defender + mad cuz bad + karen + log off + go commit die + braindead + sub bot + failed school + get a life (that doesn't suck) + blocked + pedo + zoo + you are going to brazil + can't touch this + retarded + splish splash your opinion is trash + opinion rejected + genshin player + league of legends player + counter ratio in advance + get sticked bug LOL + talk nonsense + trump supporter + your’re a full time discord mod + grammar issue + nerd + clapped + + lorem ipsum dolor sit amet + bleach + lol + done for + don't give a damn + get a job + sus + baka + sussy baka + get blocked + mad free + freer than air + furry + rip bozo + you're a (insert stereotype) + reported + ask candice + ez clap + straight cash again + idgaf + ratio again + stay angry + read FAQ + you're lost + you “re” + stay pressed + reverse double take back + pedophile + cancelled again + your dad + slight_smile + aired + cringe again + Super Idol的笑容 + yb better + i won
 
 Did you fucking get meme'd on as a cultist by a Chaplain, massive kek. Fucking get over yourself with this PR

**Comment by Maia-J** [comment]
> this PR is for council's easy access and code review, not public opinion
> 
> the reactions are pretty funny though

Yea if it's not for public opinion than why not just directly commit to the repo?

Also what did you expect from a publicly available controversial PR, a handshake?
Pat on the back maybe?

**Comment by neeshacark** [comment]
now that the vote has failed im firmly against this.

**Comment by ynot01** [comment]
Vote failed

**Pull request closed without merging**